internal/validator/cosignvalidator/cosign_validator.go

```diff
@@ -275,15 +314,15 @@ func (cv *CosignValidator) setupOptions(
 	ctx context.Context,
 	args policy.RuleOptions,
 	img *image.Image,
-	verifier sig.Verifier,
 ) (*cosign.CheckOpts, error) {
-	var ignoreTlog bool
-	if args.VerifyTLog == nil {
-		ignoreTlog = false
-	} else {
-		ignoreTlog = !*args.VerifyTLog
+	getDefault := func(b *bool) bool {
+		if b == nil {
+			return false
+		}
+		return !*b
 	}
-	opts := &cosign.CheckOpts{SigVerifier: verifier, IgnoreTlog: ignoreTlog}
+	ignoreTlog, ignoreSCT := getDefault(args.VerifyTLog), getDefault(args.VerifySCT)
```

Comment:
A function `getDefault` returns the negated bool of its input (even ignoring the nil stuff)? 🤨 

Maybe along the lines of
```
		boolOrDefault := func(b *bool, default bool) bool {
		if b == nil {
			return default
		}
		return b
		
		verifyTlog,verifySCT := boolOrDefault(args.VerifyTLog, true), boolOrDefault(args.VerifySCT, true)
                opts := &cosign.CheckOpts{IgnoreTlog: !verifyTlog, IgnoreSCT: !verifySCT}
```
?
That has reasonable naming and would work for `false` defaults, too. If you don't like the premature generalization, you could also make it `boolOrTrue` and rm the default param

Reply:
As a nice benefit, this also negates all the `if !ignoreX` below back to `if verifyX`